extensions/wikia/DataWarehouse/DataWarehouseEventProducer.class.php

```diff
@@ -287,17 +283,13 @@ public function setGeoContinent ( $continent ) {
 	}
 
 	public function setCityId ( $city_id ) {
```

Comment:
You can also use type hinting in method declaration and PHP will cast the parameter to int.

```suggestion
	public function setCityId ( int $city_id ) {
```